django_opentracing/tracer.py

```diff
@@ -64,7 +65,7 @@ def _apply_tracing(self, request, view_func, attributes):
         '''
         # strip headers for trace info
         headers = {}
-        for k,v in request.META.iteritems():
+        for k,v in listitems(request.META):
```

Comment:
again, stick with `iteritems` here?

Reply:
well we can't stick with `iteritems` because `iteritems` doesn't exist in Python 3. The simplest approach I've used is just using `.items()` because:
* Python3 still uses iterators
* it's not a big deal to not have an iterator because it's not that big
* Only Python2 has to deal with that